docs/doc-requirements.txt

```diff
@@ -0,0 +1,3 @@
+netCDF4
```

Comment:
I included an extra `.txt` file because getting the docs to build required several extra requirements I didn't have. I can either clean this up / add to it if folks have suggestions, or remove it if people prefer not to have it.